data.sql

```diff
@@ -1,5 +1,26 @@
-/* Populate database with sample data. */
+/* Populate database with sample data. */ 
+
+ INSERT INTO animals (id,name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1,'Agumon', 'Feb 03,2020', 0, true, 10.23);
+ INSERT INTO animals (id,name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2,'Gabumon', 'Nov 15, 2018', 2, true, 8);
+ INSERT INTO animals (id,name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3,'Pikachu', 'Jan 07, 2021', 1, false, 15.04);
+ INSERT INTO animals (id,name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4,'Devimon', 'May 12, 2017', 5, true, 11);
+
+/* New data */
+
+INSERT INTO animals (id,name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2,'Gabumon', 'Nov 15, 2018', 2, true, 8);
```

Comment:
Great job @AhmadiGu 🤝

- [ ] You may remove this first insert input as it is not specified within the project requirements.